src/ansys/geometry/core/connection/client.py

```diff
@@ -181,6 +183,13 @@ def __init__(
             elif grpc_backend_type == GRPCBackendType.LINUX_DMS:
                 backend_type = BackendType.LINUX_SERVICE
 
+        # backend version
+        if hasattr(grpc_backend_response, "version"):
+            ver = grpc_backend_response.version
+            self._backend_version = f"{ver.major_release}.{ver.minor_release}.{ver.service_pack}"
+        else:
+            logger.warning("The backend version is only available after 24.1 version.")
+            self._backend_version = "24.1.0"
         # Store the backend type
         self._backend_type = backend_type
```

Comment:
I would reorder this section so that the "backend_type" logic is all together, and the backend version logic comes right afterwards